apps/omg_watcher/lib/application.ex

```diff
@@ -71,7 +73,71 @@ defmodule OMG.Watcher.Application do
           depositor: %{sync_mode: :sync_with_coordinator},
           exit_processor: %{sync_mode: :sync_with_root_chain},
           exit_finalizer: %{sync_mode: :sync_with_coordinator},
-          exit_challenger: %{sync_mode: :sync_with_root_chain}
+          exit_challenger: %{sync_mode: :sync_with_root_chain},
```

Comment:
modifications to the Watcher weren't supposed to be in this PR, why? (not only this is done elsewhere, but also appears incorrect - in flight exits are handled by `ExitProcessor` for the Watcher).

In fact, all of the modifications to `omg_watcher` look like they can and should be undone, except the tests and `libsecp...`